ignite/cmd/network.go

```diff
@@ -124,17 +115,9 @@ func (n NetworkBuilder) Network(options ...network.Option) (network.Network, err
 		}
 	}
 
-	options = append(options, network.CollectEvents(n.ev))
```

Comment:
Why don't we keep this here?

Reply:
Looks like it's moved to: https://github.com/ignite-hq/cli/pull/2411/files#diff-4c5fea64848957e39a5ea392530f05bad21c37580482fe23db193603fa3099f1R40

Reply:
Yeah. I think we can keep it here. Previously event bus was initialised inside of `NetworkBuilder` constructor. Now event bus is being initialised inside of `Session`. I think we can pass eventBus to the `NetworkBuilder` as an option